element_session/export/nwb.py

```diff
@@ -74,44 +61,41 @@ def session_to_nwb(
 
     session_identifier = {
         k: v.isoformat() if isinstance(v, datetime.datetime) else v
-        for k, v in session_key.items()
-    }
+        for k, v in session_key.items()}
 
     nwbfile_kwargs = dict(
-        session_id="_".join(session_identifier.values()), identifier=str(uuid4()),
-    )
+        session_id="_".join(session_identifier.values()), identifier=str(uuid4()))
 
```

Comment:
I am following the [black](https://github.com/psf/black) guide strictly. I think this can lead to a few counter-intuitive choices but I would highly recommend it. Of course if you do not want to follow this you are free to reformat as you see fit.

Reply:
Sorry if I introduced any formatting clutter as I revised. Doing my best to stay consistent with what our team has internally. Feel free to keep black standards for now. This can be addressed after your work has concluded